charts/harbor/templates/database/database-ss.yaml

```diff
@@ -45,7 +45,7 @@ spec:
         image: {{ .Values.database.internal.image.repository }}:{{ .Values.database.internal.image.tag }}
         imagePullPolicy: {{ .Values.imagePullPolicy }}
         command: ["/bin/sh"]
-        args: ["-c", "[ -e /var/lib/postgresql/data/postgresql.conf ] && [ ! -d /var/lib/postgresql/data/pgdata ] && mkdir -m 0700 /var/lib/postgresql/data/pgdata && mv /var/lib/postgresql/data/* /var/lib/postgresql/data/pgdata/ || true"]
+        args: ["-c", "[ -e /var/lib/postgresql/data/postgresql.conf ] && [ ! -d /var/lib/postgresql/data/pgdata ] && mkdir -m 0700 /var/lib/postgresql/data/pgdata && mv /var/lib/postgresql/data/* /var/lib/postgresql/data/pgdata/ || chmod -R 0700 /var/lib/postgresql/data || true"]
```

Comment:
This operation is disallowed when `runAsNonRoot` is active, and breaks startup of harbor. As you can see I changed the line of code 2 days ago. So why do you start this PR without consulting me?

Reply:
If there is a bug with harbor we should report it

Reply:
please read:  goharbor/harbor-helm#444

Reply:
Bart reported this issue to me and we were able to fix it by editing stateful . The fix seems to work.

You are not the only one working here. You almost never consult what you are doing with the team and do not follow the review process. Would be great if could set a good example first instead of sprawling negative energy everywhere.

Reply:
https://github.com/goharbor/harbor-helm/issues/444#issuecomment-866970608

Reply:
not working with gatekeeper turned on and runAsUser enforced...

Reply:
If there is no other way you could add the annotation to bypass that policy, then you can use root to change the permissions...

Reply:
> @j-zimnowoda can I ask you to add the missing annotations to this PR?

yes, of course, I will do so